2017-10/agenda.md

```diff
@@ -0,0 +1,46 @@
+# Node.js Collaborator Summit, 2017-10-06 - 2017-10-07, Vancouver, BC, Canada
+
+**This is a DRAFT proposed agenda**
+
+## 2017-10-06 Day One
+
+|    Time Slot    | Room |      Session     |        Description       |
+| --------------- | ---- | ---------------- | ------------------------ |
+|  9:00a - 11:00a | TBD  | Core Plenary     | All-Collaborator Plenary |
+| 11:00a - 12:00p | TBD  | TBD              | TBD                      |
+| 12:00p - 1:00p  | TBD  | Lunch            | Lunch                    |
+|  1:00p - 4:00p  | TBD  | WG Sessions      | Working Group Sessions   |
```

Comment:
Let's not call them WG Sessions. Just "sessions". They don't have to be working groups and there's no need for people to have to think about whether or not they are a member and whether or not that matters. What matters is the topic and whether they are interested and invested in it. Even if working groups are meeting, don't call it that. It's not the Diagnostics WG Session. It's the Diagnostics Session.

Reply:
Ok

Reply:
+1

Reply:
@Trott There may be space for both. I agree that, in spirit, I expect there to be more ad hock collaboration sessions. I'd love to see us evaluate if these ad hoc interest groups should represent an ongoing interest group that needs regular engagement and formal WG status.